File: hIndex/hIndex.js
function solution(citations) {
  citations.sort((a, b) => {
    return a - b;
  });

  for (let i = 0; i < citations.length; i++) {
    if (citations[i] >= citations.length - i) {
      return citations.length - i;
    }
  }
  return 0;
}

console.log([3, 0, 6, 1, 5]);

File: hIndex/hIndex.py
def solution(citations):
  # 사이테이션을 정렬
  citations.sort()
  # 
  for i in range(len(citations)):
    if citations[i] >= len(citations) - i: 
      return len(citations)-i 


print(solution([3,0,6,1,5]))